sdk/storage/azure-storage-file-datalake/azure/storage/filedatalake/_data_lake_service_client.py

```diff
@@ -224,7 +225,7 @@ def get_user_delegation_key(
     @distributed_trace
     def list_file_systems(
         self, name_starts_with: Optional[str] = None,
-        include_metadata: Optional[bool] = None,
+        include_metadata: bool = False,
```

Comment:
Was this because the generated code only takes a `bool`? This is technically a breaking change to the public API, but we may be able to get away with it if `None` and `False` are treated the same in the generated code.

Reply:
It looks like the call to `self._blob_service_client.list_containers(...)` takes in `include_metadata` as well, and its type signature is:

```python3
def list_containers(
    self, name_starts_with: Optional[str] = None,
    include_metadata: bool = False,
    **kwargs: Any
) -> ItemPaged[ContainerProperties]:
```

Reply:
What about the generated code call that `self._blob_service_client.list_containers(...)` makes? If the generated code treats `None` and `False` the same, it is fine to change but if they are different it may not be.